test/src/test/java/org/corfudb/integration/StreamingIT.java

```diff
@@ -1114,6 +1115,99 @@ public void testClientExceptions() throws Exception {
         assertThat(shutdownCorfuServer(corfuServer)).isTrue();
     }
 
+    /**
+     * Test subscribe streaming API for which only streams tags are specified.
+     *
+     * (1) Test the case where tables with a common tag are opened in advance and by subscribing to the tag,
+     * updates for all tables are received.
+     *
+     * (2) Test the case where tables have been registered on a different store, and we attempt to
+     * subscribe without opening the tables (this should fail, as key, value & metadata schemas are required).
+     *
+     * (3) Test the case where partial tables belonging to a tag have been opened by the time of subscription (should still fail).
```

Comment:
Do we need to mention that all tables need to be opened in the API java doc comment?

Reply:
added